test_add_two_negative (danyuanceshi.TestMathMethod)
两个负数相加 ... ok
test_add_two_positive (danyuanceshi.TestMathMethod)
两个正数相加 ... ok
test_add_two_zero (danyuanceshi.TestMathMethod)
两个0相加 ... FAIL

======================================================================
FAIL: test_add_two_zero (danyuanceshi.TestMathMethod)
两个0相加
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\work\practice\ningmeng\unit\danyuanceshi.py", line 37, in test_add_two_zero
    self.assertEqual(1, res)
AssertionError: 1 != 0

----------------------------------------------------------------------
Ran 3 tests in 0.001s

FAILED (failures=1)
